src/components/dev-hub/section-header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { colorMap, fontSize, screenSize } from './theme';
+import { P } from './text';
+
+const SectionHeader = styled('div')`
```

Comment:
Here are the specs for the text, I think we may want to use another `P` variant here to match since the default is 16px size and 24px line height. (`letter` is `letter-spacing`)

![Screen Shot 2020-05-19 at 10 42 01 AM](https://user-images.githubusercontent.com/9064401/82340605-6aece480-99bd-11ea-82d6-3d7e24cc4acc.png)

Reply:
There's another consideration here - screen readers and search engines use the headers, `h1`, `h2`, etc to group the page and organize by importance. It would be good to keep these in importance order throughout the pages